sklearn/tree/_criterion.pyx

```diff
@@ -963,6 +964,99 @@ cdef class MSE(RegressionCriterion):
         impurity_right[0] /= self.n_outputs
 
 
+cdef class MAE(RegressionCriterion):
```

Comment:
I am surprised not to see `node_value` being overridden. The impurity criterion used at split time and the predicted value go hand in hand. In this case, one should predict the median,